src/postgraphql/withPostGraphQLContext.ts

```diff
@@ -167,6 +172,11 @@ async function setupPgClientTransaction ({
     localSettings.set(`jwt.claims.${key}`, jwtClaims[key])
   }
 
+  // If we have headers!
+  for (const key of Object.keys(reqHeaders)) {
+    localSettings.set(`req.headers.${key}`, reqHeaders[key])
```

Comment:
Is this safe?  do we use localSettings in a way where we do not expect escaping?  Are these all type-cast to text?